src/App.scss

```diff
@@ -118,6 +118,7 @@
       flex-grow: 1;
       display: flex;
       flex-direction: column;
+      overflow-y: scroll;
```

Comment:
Let's see a demo for this change and we need to test the change in Edge browser.